ServerCore/Pages/Teams/Create.cshtml

```diff
@@ -50,11 +50,13 @@ else
             }
             else
             {
+                <!--
```

Comment:
It'd be nice to have config for this so we don't forget about it next year

Reply:
I don't know how to do it without database changes and I don't want to deploy any database changes during active registration time.